general/jobSubmission/initiateDataFolders.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# create default data directories in given location
+
+set -e
+
+# one argument: directories location
+
+cd $1
+
+mkdir 0_metadata 1_raw 2_trimmed 3_aligned 4_catMemes
```

Comment:
This is great, I like how it is all on one line, efficient. I forgot you could do that.